apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.ts

```diff
@@ -0,0 +1,89 @@
+import ERR from 'async-stacktrace';
+import { Router } from 'express';
+import error = require('@prairielearn/error');
+import { logger } from '@prairielearn/logger';
+import sqldb = require('@prairielearn/postgres');
+import { QuestionAddEditor } from '../../lib/editors';
+import fs = require('fs-extra');
+import async = require('async');
+import { QuestionsPage } from './instructorQuestions.html';
+import { QuestionsPageDataAnsified, selectQuestionsForCourse } from '../../models/questions';
+
+const router = Router();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+router.get('/', function (req, res, next) {
```

Comment:
Bonus points if this is rewritten to not use `async.series` 🙂

Reply:
done